### PR TITLE
Make VAD grace period configurable for LADSPA plugin.

### PR DESCRIPTION
**Pull request by THS-on**
This should implement #47.
I've tested this with my PipeWire + Carla setup but not directly on PulseAudio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ pactl list sources short
 Then, create the new device using:
 ```sh
 pacmd load-module module-null-sink sink_name=mic_denoised_out rate=48000
-pacmd load-module module-ladspa-sink sink_name=mic_raw_in sink_master=mic_denoised_out label=noise_suppressor_mono plugin=/path/to/librnnoise_ladspa.so control=50
+pacmd load-module module-ladspa-sink sink_name=mic_raw_in sink_master=mic_denoised_out label=noise_suppressor_mono plugin=/path/to/librnnoise_ladspa.so control=50,
```
Comment (werman) [inline README.md:41]: `control=50,`

Would the lack of `,` in existing installations pose a problem?

Comment (THS-on) [reply README.md:41]: Unfortunately yes.
If the `,` is missing you get the error message:
```
Not enough control values passed, 2 expected, 1 passed.
Failed to parse, validate or set control parameters
```

Comment (THS-on) [reply README.md:41]: If we don't want to break exsiting setups we could introduce something like `noise_suppressor_vad_mono`.

Comment (werman) [reply README.md:41]: Ugh, going through https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/blob/master/src/modules/module-ladspa-sink.c - I don't see any workarounds. What a great usability... How is a new parameter is supposed to be added to a plugin.

I don't like generating another shared lib either. Hmmmmm

**Comment by werman** [comment]
> Would be great to see this merged! Particularly because the configuration concerns listed don't apply to Pipewire. What if this was merged with the caveat that if you want to modify the VAD grace period, you need to run Pipewire and let Pipewire emulate Pulse audio?

I'm making a big update in `develop` branch, which includes configurable VAD. There are probably a few things left to tweak, so I would make the release in a coming week.

**Comment by THS-on** [comment]
@werman awesome to see new development for this plugin. I use it daily with PipeWire filter chains and it just works.

If the API for the LADSPA plugin changes, do you think dropping the `lib` prefix in the name makes sense or does it cause to much confusion?

**Comment by werman** [comment]
Published 1.01 release, this MR is now obsolete.

> If the API for the LADSPA plugin changes, do you think dropping the lib prefix in the name makes sense or does it cause to much confusion?

Probably would cause confusion. Left it as is.

**Pull request closed without merging**